class-unlist-posts.php

```diff
@@ -127,8 +127,8 @@ function post_navigation_clause( $where ) {
 			$hidden_posts = get_option( 'unlist_posts', array() );
 
 			// bail if none of the posts are hidden or we are on admin page or singular page.
-			if ( ( is_admin() && ! wp_doing_ajax() ) || in_array( get_the_ID(), $hidden_posts, true ) || empty( $hidden_posts ) ) {
-				return $where;
+			if ( ( is_admin() && ! wp_doing_ajax() ) || empty( $hidden_posts ) ) {
+					return $where;
```

Comment:
This extra space can be removed, same for the condition below.

Reply:
Done